experiments/standalone/Snow/snowmip_simulation.jl

```diff
@@ -89,6 +110,29 @@ S = [parent(sol.u[k].snow.S)[1] for k in 1:length(sol.t)];
 U = [parent(sol.u[k].snow.U)[1] for k in 1:length(sol.t)];
 t = sol.t;
 
+start_day = 1
+days = start_day .+ floor.(t ./ 3600 ./ 24)
+doys = days .% 366 # doesn't account for leap year
```

Comment:
why 366 and not 365?

Reply:
I wanted day 365 to correspond to doy = 365, but I think you're right to use 365, 366 messes up the doy calculation